docs/configuration-reference/components/prometheus-operator.md

```diff
@@ -19,13 +13,13 @@ instances.
 
 * A Lokomotive cluster with a
   [PersistentVolume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) plugin, e.g.
-  [OpenEBS](openebs-operator.md) or one of the
+  [OpenEBS](../openebs-operator) or one of the
   [built-in](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#types-of-persistent-volumes)
   plugins.
 
-* An ingress controller such as [Contour](contour.md) for HTTP ingress.
+* An ingress controller such as [Contour](../contour) for HTTP ingress.
```

Comment:
This change breaks links in GitHub rendered markdown. How can we preview the changes to ensure things are not broken?

Reply:
The idea with the docs portal is that it will be used primarily for reading docs from now on. Unfortunately the links don't use `.md` in the portal.

Reply:
Okay, so as a contributor to the docs, how I can see that my changes looks correct?

Reply:
this is a gap between hugo rendering and github linking. 
As for previewing the rendered docs, i'm sure we'll smooth it out, or even better, have a canary netflify deployment for review in the github bots

Reply:
Yes, I think we should add a GitHub check in each PR linking to a netlify preview. We still need some work to get that set up though.

Reply:
Even manual instruction how to test them would be sufficient for now, but I think it should be documented.

Reply:
As discussed OOB, we've agreed on a way to test the final rendering internally. Contributors can work against the raw Markdown and we'll check if the rendering is correct, we expect most contributions won't need to check the final rendering. We'll later follow up with a way for contributors to see the rendered docs.